java_tools/gcc_map_reader/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+defaultTasks 'jar'
+
+dependencies {
+    compileClasspath fileTree( dir: 'lib', includes: [ 'junit.jar', 'annotations.jar' ] )
+}
+
+/*
+  The deprecated now building by Ant script put the
+  generated classes into folder 'build/classes/<package-name>'.
+  But Gradle by default put the generated classes to
```

Comment:
Right, but who cases about those classes location, why do we need legacy class location why not change?